lib/bolt/plugin/module.rb

```diff
@@ -159,17 +155,9 @@ def process_params(task, opts)
         # handled previously. That may not always be the case so filter them
         # out now.
         meta, params = opts.partition { |key, _val| key.start_with?('_') }.map(&:to_h)
+        params = @config ? config.merge(params) : params
```

Comment:
I _believe_ `config` will always be set. If not (perhaps in tests?) we should set a default in the constructor so that this can always just be `config.merge(params)`

Reply:
I think it should be safe to just have it be `config.merge(params)` since `config` is a required param in the initializer, so if there ever isn't config passed it will error before reaching this point.